keras/datasets/boston_housing.py

```diff
@@ -19,19 +19,15 @@ def load_data(path='boston_housing.npz', test_split=0.2, seed=113):
     path = get_file(path,
                     origin='https://s3.amazonaws.com/keras-datasets/boston_housing.npz',
                     file_hash='f553886a1f8d56431e820c5b82552d9d95cfcb96d1e678153f8839538947dff5')
-    f = np.load(path)
-    x = f['x']
-    y = f['y']
-    f.close()
+    with np.load(path) as f:
+        x, y = f['x'], f['y']
```

Comment:
In general, it is in better style to have separate statements for things like this (at least that's the style we're adopting in this codebase)

Reply:
I didn't do for the style improvement, but to sync the style across NLP datasets (as PR title says). Even 'with' is primarily for that reason (although improved robustness is nice). Compare to:

https://github.com/fchollet/keras/blob/master/keras/datasets/imdb.py#L54
https://github.com/fchollet/keras/blob/master/keras/datasets/imdb.py#L94

https://github.com/fchollet/keras/blob/master/keras/datasets/reuters.py#L53
https://github.com/fchollet/keras/blob/master/keras/datasets/reuters.py#L81

this is also the style in cifar10, cifar100, due to load_batch()
https://github.com/fchollet/keras/blob/master/keras/datasets/cifar10.py#L30

but also in mnist, simply because:
https://github.com/fchollet/keras/blob/master/keras/datasets/mnist.py#L19

it's also pervasive throughout examples.